modules/machine-health-checks-about.adoc

```diff
@@ -29,8 +29,46 @@ Consider the timeouts carefully, accounting for workloads and requirements.
 
 To stop the check, remove the resource.
 
-For example, you should stop the check during the upgrade process because the nodes in the cluster might become temporarily unavailable. The `MachineHealthCheck` might identify such nodes as unhealthy and reboot them. To avoid rebooting such nodes, remove any `MachineHealthCheck` resource that you have deployed before updating the cluster.
-However, a `MachineHealthCheck` resource that is deployed by default (such as `machine-api-termination-handler`) cannot be removed and will be recreated.
+
+[id="machine-health-checks-pause_{context}"]
+== Pausing machine health checks
+During the upgrade process, nodes in the cluster might become temporarily unavailable. The `MachineHealthCheck` might identify such nodes as unhealthy and reboot them. To avoid rebooting such nodes, pause the `MachineHealthCheck` resource before updating the cluster.
```

Comment:
There can be multiple MHC resources, so maybe better use plural?
`..., pause all MachineHealthCheck resources ...`